src/utils/utils.tsx

```diff
@@ -38,7 +38,11 @@ export const getGeography = (city: string[] | null | undefined, state: string[]
       return geo.trim() + ", "
     }
     else {
-      return geo.join(", ") + ", ";
+      var geoTrimmed = []
+      for (let i = 0; i < geo.length; i++) {

```

Comment:
Consider using a map function here:

`
const geoTrimmed = geo.map((g) => g.trim())
`

Reply:
oh shit that makes sense
 ok done